src/System.Private.CoreLib/shared/System/IO/MemoryStream.cs

```diff
@@ -299,7 +300,7 @@ public virtual int Capacity
                     }
                     else
                     {
-                        _buffer = null;
+                        _buffer = Array.Empty<byte>();
```

Comment:
Will this conflict with: https://github.com/dotnet/coreclr/pull/23700? -- If so just remember to revert before merging so that we don't have to fix merge conflicts when pulling master to the feature branch.

Reply:
> Will this conflict with

Yes, though I would hope git could automatically resolve the conflict as they're identical changes.

> If so just remember to revert before merging so that we don't have to fix merge conflicts when pulling master to the feature branch.

I'd prefer to fix it here as well and then deal with resolving the conflict.  You'd prefer otherwise?

Reply:
No that is fine. I would also hope that git would be smart enough to resolve the conflict. If not, it is not a big deal to resolve the conflict, leave as is 😄